webhook/index.js

```diff
@@ -88,12 +111,26 @@ router.post('/api/chat/', async function(req, res) {
             res.send(twiml.toString());
         });
     }
-
-    // Intents with static response handled from dialogflow console
-    else {
-        const message = twiml.message(responseText);
-        res.send(twiml.toString());
+    else if (dialogflowResponse.intent.displayName === 'register_yourself') { // Register Yourself Intent
+        responseText = await userController.addPatientInfoIntent(dialogflowResponse.queryResult, body);
+    }
+    else if (dialogflowResponse.intent.displayName === 'check_patient_profile') { // Check single patient // Needs more work
+        responseText = await userController.show(dialogflowResponse.queryResult, body);
+    }
+    else if (dialogflowResponse.intent.displayName === 'list_of_patients') { // Complete list fo all patients
+        responseText = await userController.liste(dialogflowResponse.queryResult, body);
+    }
+    else if (dialogflowResponse.intent.displayName === 'register_another_patient') { // Register a new Patient
+        responseText = await userController.newPatientIntent(dialogflowResponse.queryResult, body);
     }
+    else if (dialogflowResponse.intent.displayName === 'user_details') { // New User Intent
+        responseText = await userController.newUserIntent(dialogflowResponse.queryResult, body);
```

Comment:
While testing, I get `await` should only be used inside an async function. Please test your modules on your local by calling them in the ps_controller.js root (/) path function.